pkgs/development/python-modules/pyqtwebengine/default.nix

```diff
@@ -42,49 +41,19 @@ in buildPythonPackage rec {
     qtwebengine
   ];
 
-  propagatedBuildInputs = [ pyqt5 ]
-    ++ lib.optional (!isPy3k) enum34;
+  propagatedBuildInputs = [ pyqt5 ];
 
   dontWrapQtApps = true;
 
-  configurePhase = ''
-    runHook preConfigure
-
-    mkdir -p "$out/share/sip/PyQt5"
-
-    # FIXME: Without --no-dist-info, I get
-    #     unable to create /nix/store/yv4pzx3lxk3lscq0pw3hqzs7k4x76xsm-python3-3.7.2/lib/python3.7/site-packages/PyQtWebEngine-5.12.dist-info
-    ${python.executable} configure.py -w \
-      --destdir="$out/${python.sitePackages}/PyQt5" \
-      --no-dist-info \
-      --apidir="$out/api/${python.libPrefix}" \
-      --sipdir="$out/share/sip/PyQt5" \
-      --pyqt-sipdir="${pyqt5}/share/sip/PyQt5" \
-      --stubsdir="$out/${python.sitePackages}/PyQt5"
+  dontConfigure = true;
```

Comment:
Is this needed?

Reply:
It is, otherwise it will try to run qmake

Reply:
Can you add a comment stating that?

Reply:
Done